bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -281,7 +283,7 @@ async function incSyncMeetings(dataStore,client,context,latestSynchronizationTim
         next_page_token = json.next_page_token ?? "";
         const data = json.meetings;
         let meetingsData = data.filter(meeting => {
-            if (moment(meeting.created_at).isAfter(latestSynchronizationTime)) {
+            if (moment(meeting.created_at).isAfter(latestSynchronizationTime) && (meeting.type == 2 || meeting.type == 8) ) {
```

Comment:
Nit: Personally, I would switch conditions just for small improvement (we don't have to create a moment instance).
But not a big issue.

```suggestion
            if ((meeting.type == 2 || meeting.type == 8) && moment(meeting.created_at).isAfter(latestSynchronizationTime)) {
```

Reply:
Change has been implemented.

Reply:
👍🏼